### DISCUSSION TITLE
PHP Foundation

### DISCUSSION THREAD

**Discussion by krileon** [joomla/joomla-cms#36090] [General]
With Nikita stepping down there are efforts to form an official PHP Foundation. It appears these efforts were successful. It's sad to see Nikita go in the PHP world, but this is an amazing step forward for PHP. The language that drives Joomla and much of the web. Below is a full article about what's going on.

https://blog.jetbrains.com/phpstorm/2021/11/the-php-foundation/

I'm starting this discussion, because I think this could be a good opportunity for the Joomla foundation to get involved in. Primarily by contributing. Being a contributor would help show the development world that Joomla stands with PHP and will grow with PHP. To be blunt this could be a solid chance to put Joomla back in the minds of developers.

https://opencollective.com/phpfoundation#category-CONTRIBUTE

Figured this was at least worth discussing openly.

**Comment by alikon** [comment]
Many thanks for reporting here.
